examples/alhasan_work_area/missing_element.py

```diff
@@ -0,0 +1,10 @@
+def missing_element(lst):
+    expected_sum_of_list = 0
+    actual_sum_of_list = 0
+    for i in range(lst[0], lst[-1]+1):
+        expected_sum_of_list += i
+    for n in lst:
```

Comment:
Please rename "n" OR put an inline comment that tells the devs, what n could be. Is it an integer coming from lst or something else ?